### PR TITLE
Update the documentation according to the ldap improvement

### PR DESCRIPTION
**Pull request by MaelAudren**

### CODE REVIEW

File: src/docs/user/ProActiveUserGuide.adoc [hunk 1]
```diff
@@ -1562,16 +1562,20 @@ The next parameters are provided as task variables:
...
-|ldapSearchBase|base domain name for LDAP queries|`dc=yourOrganization,dc=com`
+|ldapDnBase|The base distinguished name of the catalog| `dc=yourOrganization,dc=com`
+|ldapUsername|The username | `cn=admin` for global admin or `cn=yourName,ou=users` for a specific user
+|ldapSearchBase|base domain name for ldap queries|`dc=specificDepartment` or can be empty for base DN search
 |ldapSearchFilter|LDAP search filter|`(objectclass=*)`
 |ldapSelectedAttributes|list of attributes to retrieve|`attributeName1,attributeName2`
+
 |===
 
+If any of these variables must stay private you'll need to set them as third-party credentials (details are given below) instead.
```
Comment (marcocast) [inline src/docs/user/ProActiveUserGuide.adoc]: without "instead"

File: src/docs/user/ProActiveUserGuide.adoc [hunk 2]
```diff
@@ -1562,16 +1562,20 @@ The next parameters are provided as task variables:
...
+
 ==== Authenticate with Third-party credentials
 
-In order to be able to authenticate to the LDAP server you need to define the right Scheduler user in the <<_third_party_credentials,Third-party credentials>>: 
+In order to be able to authenticate to the LDAP server you need to define for the right Scheduler user in the <<_third_party_credentials,Third-party credentials>>:
```
Comment (marcocast) [inline src/docs/user/ProActiveUserGuide.adoc]: why "for"?

File: src/docs/user/ProActiveUserGuide.adoc (hunk 1 above)
Comment (MaelAudren) [inline src/docs/user/ProActiveUserGuide.adoc]: Suggestion applied

File: src/docs/user/ProActiveUserGuide.adoc (hunk 2 above)
Comment (MaelAudren) [inline src/docs/user/ProActiveUserGuide.adoc]: Suggestion applied

Comment (MaelAudren) [inline src/docs/user/ProActiveUserGuide.adoc]: It was not relevant to display a list for one item so I replace the sentence by naming the item.

**Pull request merged**